app/logic/Mail/ActiveSync/ActiveSyncAccount.ts

```diff
@@ -366,7 +366,7 @@ export class ActiveSyncAccount extends MailAccount {
     }
   }
 
-  protected async makeRequest(command: string, request: any, callback?: (response: any) => Promise<void>): Promise<any> {
+  protected async makeRequest(command: string, request: any, callback?: (response: any, resync?: true) => Promise<void>): Promise<any> {
```

Comment:
Please do not add edge case parameters to very generic functions.
See a suggestion below on how to fix it.